### DISCUSSION TITLE
Can't get a windowrulev2 to work

### DISCUSSION THREAD

**Discussion by Kommynct** [hyprwm/Hyprland#5085] [Q&A] [answered]
I have a windowrule to pin a window, it seems to do nothing

`windowrulev2 = pin,class:(chromium),title:(MAX - Chromium)`

I've double checked hyprctl clients and copy/pasted from them to make this rule, I suspect this is a bug

```
Window 55efb2536f80 -> MAX - Chromium:
	mapped: 1
	hidden: 0
	at: 4455,150
	size: 300,759
	workspace: 2 (2:    )
	floating: 1
	monitor: 1
	class: chromium
	title: MAX - Chromium
	initialClass: chromium
	initialTitle: Untitled - Chromium
	pid: 48448
	xwayland: 0
	pinned: 0
	fullscreen: 0
	fullscreenmode: 0
	fakefullscreen: 0
	grouped: 0
	swallowing: 0
	focusHistoryID: 5
```

**Comment by vaxerski** [comment]
pin is static so it uses the `initial` properties for class and title

**Comment by Kommynct** [comment]
Is there a workaround you would suggest considering the initialtitle here is untitled - chromium?

**Reply by vaxerski** [reply, marked as answer]
scripting is your only choice. (or, plugin)

**Reply by Kommynct** [reply]
Is it planned to make as many of the static rules dynamic as possible in the future?

**Reply by vaxerski** [reply]
maybe
